src/improvements/tasks/MultisigTask.sol

```diff
@@ -112,32 +108,12 @@ abstract contract MultisigTask is Test, Script {
     TaskConfig public config;
 
     /// @notice flag to determine if the task is being simulated
-    bool private _buildStarted;
-
-    /// @notice The address of the child multisig for this task
-    /// this state variable will only be set when function
-    /// `signFromChildMultisig` is called.
-    address public childMultisig;
+    uint256 private _buildStarted;
```

Comment:
Wy is this now a uint?

Reply:
I mentioned the reason here in this thread: https://github.com/ethereum-optimism/superchain-ops/pull/640#discussion_r1965805909